packages/components/fields/creatable-select-field/src/creatable-select-field.tsx

```diff
@@ -0,0 +1,453 @@
+  backspaceRemovesValue?: ReactSelectCreatableProps['backspaceRemovesValue'];
+  /**
+   * Map of components to overwrite the default ones, see [what components you can override](https://react-select.com/components)
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  components?: ReactSelectCreatableProps['components'];
+  /**
+   * Custom method to filter whether an option should be displayed in the menu
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  filterOption?: ReactSelectCreatableProps['filterOption'];
+  /**
+   * The id to set on the SelectContainer component
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  containerId?: ReactSelectCreatableProps['id'];
+  /**
+   * Is the select value clearable
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  isClearable?: ReactSelectCreatableProps['isClearable'];
+  /**
+   * Is the select disabled
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  isDisabled?: ReactSelectCreatableProps['isDisabled'];
+  /**
+   * Is the select read-only
+   */
+  isReadOnly?: ConstrainBooleanParameters;
+  /**
+   * Override the built-in logic to detect whether an option is disabled
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  isOptionDisabled?: ReactSelectCreatableProps['isOptionDisabled'];
+  /**
+   * Support multiple selected options
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  isMulti?: ReactSelectCreatableProps['isMulti'];
+  /**
+   * Whether to enable search functionality
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  isSearchable?: ReactSelectCreatableProps['isSearchable'];
+  /**
+   * Indicates the input field has an error
+   */
+  hasWarning?: boolean;
+  /**
+   * Maximum height of the menu before scrolling
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  maxMenuHeight?: ReactSelectCreatableProps['maxMenuHeight'];
+  /**
+   * 	Dom element to portal the select menu to
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  menuPortalTarget?: ReactSelectCreatableProps['menuPortalTarget'];
+  /**
+   * z-index value for the menu portal
+   */
+  menuPortalZIndex?: number;
+  /**
+   * whether the menu should block scroll while open
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  menuShouldBlockScroll?: ReactSelectCreatableProps['menuShouldBlockScroll'];
+  /**
+   * 	Name of the HTML Input (optional - without this, no input will be rendered)
```

Comment:
```suggestion
   * Name of the HTML Input (optional - without this, no input will be rendered)
```
Remove extra spaces.